### ISSUE TITLE
[Suggestions] Black list

### ISSUE THREAD

**Issue by Becold** [AnthonyOSX/GO-Bot#14]
I dont want to transfer my Eevee. A blacklist would be cool.
Edit: And I dont want to transfer my pokemon who have more than X CP. (If I have2 sames pokemon over 2000, i want to keep them both)

And I dont want to hatch my 10km egg on my low levels.

**Comment by iamSuperFX** [comment]
I've read that eggs are the lvl you catch them at.

**Comment by jdspringr07** [comment]
I would like the number to be based on percentage CP, not necessarily pokemon over a certain CP. Let me explain.

As you level, you catch higher level CP pokemon in the wild, but "CP 400" at lvl 19 is completely different from a "CP 400" at lvl 27. Each pokemon, caught at different levels, have a high potential base CP available to them .

A LvL 19, for a example, might be able to get an Eeevee up to "600 CP" whereas a lvl 27 could get upt o a "900 CP" (just throwing numbers out there). You can see this when you compare different pokemon caught from different levels.

TL;DR: Would be nice to keep "X Amount" of pokemon depending on the overall CP value. (i.e. 90% full CP potential)

**Comment by Becold** [comment]
Update v0.0.5 seems fixing this issue (at least for me).
